internal/saucecloud/cloud.go

```diff
@@ -932,11 +932,26 @@ func (r *CloudRunner) uploadCLIFlags(jobID string, realDevice bool, content inte
 	}
 }
 
-func (r *CloudRunner) deprecationMessage(frameworkName string, frameworkVersion string) string {
+func (r *CloudRunner) deprecationMessage(frameworkName string, frameworkVersion string, removalDate time.Time) string {
+	formattedDate := removalDate.Format("Jan 02, 2006")
+
 	return fmt.Sprintf(
-		"%s%s%s",
-		color.RedString(fmt.Sprintf("\nVersion %s for %s is deprecated and will be removed during our next framework release cycle !\n\n", frameworkVersion, frameworkName)),
+		"%s%s%s%s%s",
+		color.RedString("\n\n************************************* WARNING *************************************\n"),
```

Comment:
can you move the `"\n\n************************************* WARNING *************************************\n"` to https://github.com/saucelabs/saucectl/blob/main/internal/msg/msg.go? So it can be more readable

Reply:
That has been updated ! Thanks for the suggestion !